packages/block-editor/src/components/link-control/test/index.js

```diff
@@ -0,0 +1,527 @@
+/**
+ * External dependencies
+ */
+import { render, unmountComponentAtNode } from 'react-dom';
+import { act, Simulate } from 'react-dom/test-utils';
+import { first, last, nth } from 'lodash';
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+import { UP, DOWN, ENTER } from '@wordpress/keycodes';
+/**
+ * Internal dependencies
+ */
+import LinkControl from '../index';
+import { fauxEntitySuggestions, fetchFauxEntitySuggestions } from './fixtures';
+
+function eventLoopTick() {
+	return new Promise( ( resolve ) => setImmediate( resolve ) );
+}
+
+let container = null;
+
+beforeEach( () => {
+	// setup a DOM element as a render target
+	container = document.createElement( 'div' );
+	document.body.appendChild( container );
+} );
+
+afterEach( () => {
+	// cleanup on exiting
+	unmountComponentAtNode( container );
+	container.remove();
+	container = null;
+} );
+
+describe( 'Basic rendering', () => {
+	it( 'should display with required props', () => {
+		act( () => {
+			render(
+				<LinkControl
+				/>, container
```

Comment:
According to the [current README](https://github.com/WordPress/gutenberg/blob/cf9742c0ce56c2b10f282b2a0d35a852ada75aa2/packages/block-editor/src/components/link-control/README.md) file, there are 2 required props:

- `value`
- `className`

Shouldn't those be here?

Aside: I would assume `className` doesn't need to be required.

Reply:
> According to the [current README](https://github.com/WordPress/gutenberg/blob/cf9742c0ce56c2b10f282b2a0d35a852ada75aa2/packages/block-editor/src/components/link-control/README.md) file, there are 2 required props:
> 
> * `value`
> * `className`
> 
> Shouldn't those be here?
> 
> Aside: I would assume `className` doesn't need to be required.

The pull request at #19736 seeks to update the documentation and test description to reflect the fact there are presently no required props for the component.